docs/source/app-dev/bindings-scala/index.rst

```diff
@@ -0,0 +1,66 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Scala bindings
+##############
+
+.. toctree::
+  :hidden:
+
+The Scala bindings is a client implementation of the *Ledger API*
+based on `Akka Streams API <https://doc.akka.io/docs/akka/current/stream/index.html>`_.
+
+Introduction
+************
+
+The Scala bindings library lets you write applications that connect to the Digital Asset distributed ledger using Scala programming language.
+
+There are two main components:
+
+- The Akka Streams based API
+    The API to send commands to the ledger and receive transactions back.
+
+- Scala codegen
+    DAML to Scala code generator. Generates Scala classes from DAML models. The generated Scala code provides a type safe way of creating contracts (create command) and exercising contract choices (exercise command).
+
+The reader must be familiar with:
+
+- :doc:`DAML language </daml/reference/index>`
+- :doc:`Ledger API </app-dev/grpc/index>`
+- `Akka Streams API <https://doc.akka.io/docs/akka/current/stream/index.html>`_
+- `Scala programming language <https://www.scala-lang.org>`_
+
+Getting started with SBT
```

Comment:
I don't understand - is this describing how to set up SBT in general, or for the quickstart example specifically?

Reply:
This is SBT specific setup., there is nothing quickstart specific in here.

Reply:
I am just referring to something that I know works.